src/components/forms/OnboardingProfessionalForm.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+import { educationLevels } from "@/shared/onboarding";
+import { formSchema } from "@/shared/onboarding";
+import { FormSchema } from "@/shared/onboarding";
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Input, Label, Select, Form } from "@/shared/components";
+
+export default function OnboardingProfessionalForm() {
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<FormSchema>({
+    resolver: zodResolver(formSchema),
+  });
+  const values = watch([
+    "educationLevel",
+    "profissao",
+    "empresaOrganizacao",
+    "githubPortifolio",
+    "linkedin",
+  ]);
+
+  function onSubmit(data: FormSchema) {
+    console.log(data);
+  }
```

Comment:
We should also consider moving all this to a hook

Reply:
Next PR going to create a new PR only for form validation with specific hookk